config/locales/es.yml

```diff
@@ -827,10 +840,10 @@ es:
       reserved_namespace_html:
         one: Esta gema existió previamente, pero fue eliminada por sus propietarios.
           El equipo de RubyGems.org ha reservado este nombre por un día más. Luego
-          cualquiera podrá solicitar este nombre de gema usando `gem push`. <br>
-          Si eres el anterior propietario, puedes cambiar la propiedad de esta gema
-          usando el comando `gem owner`. También puedes crear nuevas versiones de
-          esta gema usando `gem push`.
+          cualquiera podrá solicitar este nombre de gema usando `gem push`. <br> Si
```

Comment:
Is this change intentional?

Reply:
Same as the above comment with fill-locales.